src/cache/cache.service.ts

```diff
@@ -0,0 +1,15 @@
+import { CACHE_MANAGER, Inject, Injectable } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+
+@Injectable()
+export class CacheService {
+  constructor(@Inject(CACHE_MANAGER) private cacheManager: Cache) {}
+
+  async get(key: string): Promise<any> {
+    return this.cacheManager.get(key);
+  }
+
+  async set(key: string, value: any, ttl: number): Promise<void> {
```

Comment:
` async set(key: string, value: any, ttl: number) {`
since it will not return anything

Reply:
No changes. Promise<void> is just an explicit way to tell that we are waiting for nothing. Better to make return data types explicit since we are using typescript.